### DISCUSSION TITLE
Get Type Sets_PaymentMethods not found error in graphql response message

### DISCUSSION THREAD

**Discussion by shirowayfy** [statamic/cms#5192] [Troubleshooting]
Get Type Sets_PaymentMethods not found error in graphql response message
<img width="704" alt="image" src="https://user-images.githubusercontent.com/53089407/152340766-8451db7a-0995-4ec7-ba2e-cad06e0e6169.png">

**Comment by jasonvarga** [comment]
You're going to need to provide more details than that. Your query, your blueprint, etc.
